docs/multichannel.md

```
@@ -0,0 +1,62 @@
+← [Documentation index](README.md)
+
+# Multichannel and Performance
+
+## Multichannel Support
+
+PyOctaveBand natively supports multichannel signals (e.g., Stereo, 5.1,
+Microphone Arrays) using **fully vectorized operations**. Input arrays of shape
+`(N_channels, N_samples)` are processed in parallel, offering significant
+performance gains over iterative loops.
+
+<picture><source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/jmrplens/PyOctaveBand/main/.github/images/signal_response_multichannel_dark.png"><img src="https://raw.githubusercontent.com/jmrplens/PyOctaveBand/main/.github/images/signal_response_multichannel.png" width="80%"></picture>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)